### PR TITLE
Some fixes for the anomalous crystal

### PR DESCRIPTION
**Pull request by Ergovisavi**
Just some fixes for the Anomalous crystal
* Dark Reprise crystal properly regenerates organs and limbs for revivees - And now also clears off that stupid O2 warning
* Removed all the projectiles from the emitter crystal except the colossus one because they were either completely broken (magic was completely fucked) or boring. Now it just spits out a death bolt every time, a bit more predictable and generally more all around useful
* Fixed an issue where players getting killed while possessing mobs that qdel on death weren't able to get back in their bodies

:cl:
fix: Fixed a few anomalous crystal effects
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -357,6 +357,7 @@
 		updatehealth() //then we check if the mob should wake up.
 		update_canmove()
 		update_sight()
+		clear_alert("oxy")
```
Comment (AnturK) [inline code/modules/mob/living/living.dm]: Shouldn't this be cleared automatically ?

Comment (Ergovisavi) [inline code/modules/mob/living/living.dm]: It isn't, evidently. For example, put someone in crit, turn them into an ash lizard, then revive them, they'll have an O2 alert on their screen forever.

**Comment by DoctoreMad** [comment]
>Removed all the projectiles from the emitter crystal except the colossus one because they were either completely broken (magic was completely fucked) or boring. Now it just spits out a death bolt every time, a bit more predictable and generally more all around useful

Ehhh
What is its use?

**Comment by Ergovisavi** [comment]
Death bolts destroy anything they hit. It's basically an oversized battering ram projectile.

It was in there before, it's just the only one now.

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 2]
```diff
@@ -734,7 +735,7 @@ Difficulty: Very Hard
 
 /obj/structure/closet/stasis/process()
 	if(holder_animal)
-		if(holder_animal.stat == DEAD && !QDELETED(holder_animal))
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: Why did you remove this check?

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 3]
```diff
@@ -762,7 +763,7 @@ Difficulty: Very Hard
 		L.disabilities &= ~MUTE
 		L.status_flags &= ~GODMODE
 		L.notransform = 0
-		if(holder_animal && !QDELETED(holder_animal))
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: Ditto

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm (hunk 2 above)
Comment (Ergovisavi) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: If the mob it's in gets Qdeleted, it qdeletes its contents. Closets that get qdeleted drop their contents. If it fails to fire because a mob is qdeleted, it fails to assign the ghost back to the player's corpse, IE: You get stepped on as a cockroach and go splat, your body falls out but your ghost isn't able to get back in your body.

Basically, checking if a mob is qdeleted is entirely irrelevant because it can't be qdeleted before running this code.

**Review by Cyberboss — APPROVED** [review]
